test/metabase/sync/util_test.clj

```diff
@@ -54,9 +54,8 @@
         ;; make sure both of the futures have finished
         (deref f1)
         (deref f2)
-        ;; Check the number of syncs that took place. Should be 2 (just the first)
-        (is (= 2
-               @calls-to-describe-database))))))
+        ;; Check the number of syncs that took place. Should be 1 (just the first)
```

Comment:
This comment is no more helpful than the actual check following it and I don't quite understand "just the first". I would remove the comment.